kasa/smart/smartdevice.py

```diff
@@ -309,6 +308,26 @@ async def _initialize_features(self):
             for feat in module._module_features.values():
                 self._add_feature(feat)
 
+    def has_module(self, module_name):
+        """Return true if the device has the module."""
+        if module_name in self.modules:
+            return True
+        elif self._exposes_child_modules:
+            for child in self._children.values():
+                if module_name in child.modules:
+                    return True
+        return False
+
+    def get_module(self, module_name):
+        """Return the module from the device modules."""
+        if module_name in self.modules:
+            return self.modules[module_name]
+        elif self._exposes_child_modules:
+            for child in self._children.values():
+                if module_name in child.modules:
+                    return child.modules[module_name]
+        return False
```

Comment:
This should raise an error, will update

Reply:
Or perhaps return none, so you can do something like?
```
if (mod := dev.get_module("Brightness")) is not None:
    mod.set_brightness(123)
```

Reply:
Done